### PR TITLE
Add NLP rules for scatter and general

### PR DESCRIPTION
**Pull request by PCXDME**
#48 #50

### CODE REVIEW

File: api/functions/scatterPlot.js [hunk 1]
```diff
@@ -47,11 +47,30 @@ function changeMarkerSize(size, trace) {
   return actionHelper.updateOneStyle('marker.size', size, trace);
 }
 
+// multiplier be any number
```
Comment (fhnaseer) [inline api/functions/scatterPlot.js]: Proper comments,

File: api/helpers/actionHelper.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 function updateStyles(styleUpdates, trace) {
```
Comment (fhnaseer) [inline api/helpers/actionHelper.js:7]: Can you add comments for functions in this file?,

File: api/functions/scatterPlot.js (hunk 1 above)
Comment (PCXDME) [reply api/functions/scatterPlot.js]: Done

File: api/helpers/actionHelper.js (hunk 2 above)
Comment (PCXDME) [reply api/helpers/actionHelper.js:7]: Done

**Review by fhnaseer — APPROVED** [review]

**Pull request merged**